python_file_reader.py

```diff
@@ -0,0 +1,31 @@
+def twod_read_input(mylist, file_name): #returns text from given file in a 2d list
+    f = open(file_name, 'r')
+    file_data = f.read()
+    input_text_block = file_data.strip().split('\n')
+    for line in input_text_block:
+        row = []
+        for char in line:
+            row.append(char)
+        mylist.append(row)
+
+def oned_read_input(mylist, file_name): #returns text from given file in a 1d list
+    f = open(file_name, 'r')
+    file_data = f.read()
+    input_text_block = file_data.strip().split('\n')
+    for line in input_text_block:
+        for char in line:
+            mylist.append(char)
+
+
+file_name = 'input.txt'
+
+twod_list =[]
```

Comment:
Can you run the Black formatter on the code once your done? Just so it can clean up some small syntax inconsistencies :100: